src/librustdoc/html/static/main.js

```diff
@@ -2361,6 +2360,11 @@ function defocusSearchBar() {
     }
 
     (function() {
+        var toggles = document.getElementById("toggle-all-docs");
+        if (toggles) {
+            toggles.onclick = toggleAllDocs;
+        }
```

Comment:
The change here is just so that `toggles` isn't in the global scope, right? Looks good to me if so.

Reply:
Absolutely!